modules/3_fips/files/fips.yml

```diff
@@ -10,4 +10,4 @@
     - name: Reboot the host
       become_user: root
       ansible.builtin.reboot:
-        reboot_timeout: 30
\ No newline at end of file
+        reboot_timeout: 3000
```

Comment:
Also increased reboot_timeout because tang_setup was failing for tang servers. And after increasing it got passed.